test/built-ins/Object/defineProperty/15.2.3.6-4-591.js

```
@@ -9,51 +9,61 @@ es5id: 15.2.3.6-4-591
-                return data2;
-            },
-            enumerable: false,
-            configurable: true
-        });
-
-        var meeting = Object.create(appointment);
-        var data3 = "In-person meeting";
-        Object.defineProperty(meeting, "conferenceCall", {
-            get: function () {
-                return data3;
-            },
-            enumerable: false,
-            configurable: false
-        });
-
-        var teamMeeting = Object.create(meeting);
-        teamMeeting.name = "IE Team Meeting";
-        var dateObj = new Date("10/31/2010 08:00");
-        teamMeeting.startTime = dateObj;
-        teamMeeting.conferenceCall = "4255551212";
-
-        var hasOwnProperty = !teamMeeting.hasOwnProperty("name") &&
-            !teamMeeting.hasOwnProperty("startTime") &&
-            !teamMeeting.hasOwnProperty('conferenceCall');
-
-        return hasOwnProperty && teamMeeting.name === "NAME" &&
-            teamMeeting.startTime === 1001 &&
-            teamMeeting.conferenceCall === "In-person meeting";
-    }
-runTestCase(testcase);
+var appointment = {};
+
+var data1 = 1001;
+Object.defineProperty(appointment, "startTime", {
+    get: function () {
+        return data1;
+    },
+    enumerable: false,
+    configurable: false
+});
+var data2 = "NAME";
+Object.defineProperty(appointment, "name", {
+    get: function () {
+        return data2;
+    },
+    enumerable: false,
+    configurable: true
+});
+
+var meeting = Object.create(appointment);
+var data3 = "In-person meeting";
+Object.defineProperty(meeting, "conferenceCall", {
+    get: function () {
+        return data3;
+    },
+    enumerable: false,
+    configurable: false
+});
+
+var teamMeeting = Object.create(meeting);
+
+verifyNotWritable(teamMeeting, "name", "nocheck");
+verifyNotWritable(teamMeeting, "startTime", "nocheck");
+verifyNotWritable(teamMeeting, "conferenceCall", "nocheck");
+
+try {
+    teamMeeting.name = "IE Team Meeting";
+} catch (e) {}
```

The empty catch blocks are kind of odd. And shouldn't the verifyNotWritable calls test this case?